typings/index.d.ts

```diff
@@ -2577,6 +2574,19 @@ declare module 'discord.js' {
     name?: string;
   }
 
+  interface GuildCreateOptions {
+    afkChannelID?: number;
```

Comment:
```suggestion
    afkChannelID?: Snowflake;
```
The type is `snowflake` in Discord's doc.

Reply:
Even though the docs says it's a snowflake, it can accept a number of one of the IDs used in the `channels` (like how `parent_id` works). Should I just use `Snowflake` because that's what the docs says, or should we allow (normal) numbers as well because a `Snowflake` implies that the number as to be in the snowflake format? I also created https://github.com/discord/discord-api-docs/issues/2089 that addresses this.

Reply:
Sorry, my bad. I didn't read that:
```
When using the channels parameter, the id field within each channel object may be set to an integer placeholder, and will be replaced by the API upon consumption.
```
So, I think that `number` is the suitable type.